edgelet/iotedge/src/lib.rs

```diff
@@ -16,6 +16,9 @@
     clippy::use_self
 )]
 
+#[macro_use]
+extern crate lazy_static;
```

Comment:
There is hardly ever a reason to use `extern crate` in Rust 2018. Just import `lazy_static::lazy_static` when you need to use it.